app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -55,7 +59,16 @@ class HomeFragment : Fragment() {
     }
 
     private fun initTiles() = with (tileContainer) {
-        adapter = HomeTileAdapter(onUrlEnteredListener)
+        var homeTiles = mutableListOf<HomeTile>()
+        val inputAsString = context.assets.open(HOME_TILES_JSON_PATH).bufferedReader().use { it.readText() }  // defaults to UTF-8
+        val jsonArray = JSONObject(inputAsString).getJSONArray(HOME_TILES_JSON_KEY)
+        for (i in 0..(jsonArray.length() - 1)) {
+            val url = jsonArray.getJSONObject(i).getJSONObject("tile_" + i.toString()).getString("url")
+            val title = jsonArray.getJSONObject(i).getJSONObject("tile_" + i.toString()).getString("title")
+            val imgPath = HOME_TILES_PREFIX + jsonArray.getJSONObject(i).getJSONObject("tile_" + i.toString()).getString("img")
+            homeTiles.add(HomeTile(url, title, imgPath))
```

Comment:
Add the "identifier" field to the HomeTile object and add a comment about it.

Basically, having an identifier will make it easier for people to access and check the object - could also a) make this optional, or just use the url, up to you.